cubist/tests/test.py

```diff
@@ -1,17 +1,25 @@
 import pandas as pd
```

Comment:
Is this file being used for anything? Doesn't seem like it fits with your unit tests

Reply:
moved to examples folder